lms/static/js/learner_dashboard/views/unenroll_view.js

```diff
@@ -0,0 +1,117 @@
+             HtmlUtils
+         ) {
+             return Backbone.View.extend({
+                 el: '.unenroll-modal',
+
+                 initialize: function(options) {
+                     this.urls = options.urls;
+
+                     $('.action-unenroll').click(function(event) {
+                         var $element = $(event.target),
+                             trackInfo = $element.data('track-info'),
+                             courseNumber = $element.data('course-number'),
+                             courseName = $element.data('course-name'),
+                             certNameLong = $element.data('cert-name-long'),
+                             survey, i;
+                         HtmlUtils.setHtml(
+                             $('#track-info'), HtmlUtils.HTML(window.interpolate(trackInfo, {
+                                 course_number: HtmlUtils.joinHtml(
+                                    HtmlUtils.HTML("<span id='unenroll_course_number'>"),
+                                    courseNumber,
+                                    HtmlUtils.HTML('</span>')
+                                 ),
+                                 course_name: HtmlUtils.joinHtml(
+                                    HtmlUtils.HTML("<span id='unenroll_course_name'>"),
+                                    courseName,
+                                    HtmlUtils.HTML('</span>')
+                                 ),
+                                 cert_name_long: HtmlUtils.joinHtml(
+                                    HtmlUtils.HTML("<span id='unenroll_cert_name'>"),
+                                    certNameLong,
+                                    HtmlUtils.HTML('</span>')
+                                 )
+                             }, true))
+                         );
+                         HtmlUtils.setHtml(
+                             $('#refund-info'), HtmlUtils.HTML($element.data('refund-info'))
+                         );
+                         $('#unenroll_course_id').val($element.data('course-id'));
+
+                         // Randomize survey option order
+                         survey = document.querySelector('.options');
+                         for (i = survey.children.length; i >= 0; i--) {
+                             survey.appendChild(survey.children[Math.random() * i | 0]);
+                         }
+                     });
+
+                     $('#unenroll_form').on('ajax:complete', function(event, xhr) {
+                         var errorText,
+                             reasonsSurvey = HtmlUtils.HTML($('.reasons_survey'));
+                         if (xhr.status === 200) {
+                             $('.inner-wrapper header').hide();
+                             $('#unenroll_form').after(HtmlUtils.ensureHtml(reasonsSurvey).toString());
+                             $('.reasons_survey .slide1').removeClass('hidden');
+                             $('#unenroll_form').hide();
```

Comment:
This jQuery object is defined twice. Either chain the commands or create a variable.